main/acle.md

```diff
@@ -1509,24 +1509,34 @@ SVE language extensions:
 > [The __ARM_FEATURE_SVE_BITS macro](#the-__arm_feature_sve_bits-macro)
 > for details.
 
-**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==1`**
+**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==N`**
 
-> This indicates that applying the `arm_sve_vector_bits` attribute
+> N >= 1, this indicates that applying the `arm_sve_vector_bits` attribute
```

Comment:
> \`N >= 1\` indicates

The other conditions involving N should be in backticks too, both for this macro and the one below.